### PR TITLE
Fixing (most) quest, archaeology & diary issues

### PR DESCRIPTION
**Pull request by SnowyToo**
~Diary task fixes following shortly~
E: Diary fixes implemented. Underworld diaries currently not all doable by vanilla (Kili's tasks don't show up for vanilla's + Anachronia content necessary)

### CODE REVIEW

File: rs3-chunkpicker-chunkinfo-export.json [hunk 1]
```diff
@@ -4405,14 +4445,15 @@
 				"Primary": false,
 				"Priority": 7
 			},
-			"Clear the ~|Barricade (Archaeology)|~ in the Warforge": {
+			"Clear the ~|barricade (Archaeology)|~ in the Warforge": {
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: Change to `barricade (Warforge Dig Site)` in 3 spots

File: rs3-chunkpicker-chunkinfo-export.json [hunk 2]
```diff
@@ -88423,7 +88511,6 @@
 					"~|Desert Treasure|~ Complete the quest": "Quest",
 					"~|Horror from the Deep|~ Complete the quest": "Quest"
 				},
-				"QuestPointsNeeded": 176,
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: Revert this

File: rs3-chunkpicker-chunkinfo-export.json [hunk 3]
```diff
@@ -79943,6 +80029,7 @@
 				"Kudos": 5,
 				"BaseQuest": "In Aid of the Myreque",
 				"Chunks": ["Hallows Myreque Hideout"],
+				"Reward": ["Nail beast nails"],
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: You can do better :)

File: rs3-chunkpicker-chunkinfo-export.json [hunk 4]
```diff
@@ -92296,7 +92383,6 @@
 				"Skills": {
 					"Prayer": 50
 				},
-				"KudosNeeded": 125,
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json]: You can also do better :)

File: rs3-chunkpicker-chunkinfo-export.json [hunk 5]
```diff
@@ -114338,273 +114427,273 @@
 			},
 			"Excavate any artifact": {
 				"Objects": ["AnyExcavationSpot[+]"],
-				"Chunks": ["13108, 13364, 13365"]
+				"Chunks": ["13108", "13364", "13365"]
 			},
-			"Complete the archaeolgy collection 'Anarchic Abstraction'":{
-				"Items": ["'Disorder' painting", "'The Lake of Fire' painting", "'Pandemonium' tapestry"],
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:114344]: Undo

File: rs3-chunkpicker-chunkinfo-export.json [hunk 6]
```diff
@@ -114338,273 +114427,273 @@
...
 				"Items": ["Dominarian device", "Fishing trident", "Amphora", "Rod of Asclepius", "Kopis dagger", "Xiphos short sword"],
 				"NPCs": ["Sir Atcha"]
 			},
-			"Complete the archaeolgy collection 'Saradominist IV'":{
+			"Complete the archaeology collection 'Saradominist IV'":{
 				"Items": ["'The Pride of Padosan' painting", "'Hallowed Be the Everlight' painting", "'The Lord of Light' painting", "'The Enlightened Soul' scroll", "'The Eudoxian Elements' tablet", "Doru spear", "Kontos lance"],
 				"NPCs": ["Sir Atcha"]
 			},
-			"Complete the archaeolgy collection 'Zarosian I'":{
-				"Items": ["Venator dagger", "Venator light crossbow", "Primis Elementis standard", "Legionary gladius", "Legionary square shield", "Zaros effigy", "Zarosian training dummy", "Legatus Maximus figurine", "'Solem in Umbra' painting"],
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:114476]: undo

File: rs3-chunkpicker-chunkinfo-export.json [hunk 7]
```diff
@@ -158280,8 +158381,8 @@
 				"Revenant bane off hand longsword"
 			],
 			"Barbarian bait[+]": [
-				"feathers+",
```
Comment (ThaFrenchFry) [inline rs3-chunkpicker-chunkinfo-export.json:158283]: Add `Feathers[+]`

**Review by ThaFrenchFry — CHANGES_REQUESTED** [review]

File: rs3-chunkpicker-chunkinfo-export.json (hunk 7 above)
Comment (SnowyToo) [reply rs3-chunkpicker-chunkinfo-export.json:158283]: Can't nest itemsPlus. Have added colored feathers to this list.

**Comment by SnowyToo** [comment]
Closed unintentionally

**Review by ThaFrenchFry — APPROVED** [review]

**Review by ThaFrenchFry — APPROVED** [review]

**Review by ThaFrenchFry — APPROVED** [review]

File: rs3-chunkpicker-chunkinfo-export.json [hunk 8]
```diff
@@ -308776,7 +309045,7 @@
 				"Weight": 15
 			}
 		},
-		"Raptor": {
+		"The Raptor#Fort Forinthry": {
```
Comment (SnowyToo) [inline rs3-chunkpicker-chunkinfo-export.json:309048]: Raptor does not show up on Slayer tab in the picker currently. Could this be related to the NPC being task-locked behind Unwelcome Guests, or am I just missing something obvious?

**Review by ThaFrenchFry — APPROVED** [review]

**Review by Connor3455 — APPROVED** [review]

**Review by ThaFrenchFry — APPROVED** [review]

**Review by Connor3455 — APPROVED** [review]

**Pull request merged**